openvpn: T2957: show openvpn not returning anything

marcushoff opened the pull request:
Fix location of status files directory

marcushoff committed:
Automatic PR from upstream

marcushoff committed:
Merge remote-tracking branch 'upstream/current' into current

marcushoff committed:
Merge remote-tracking branch 'upstream/current' into current

marcushoff committed:
Merge remote-tracking branch 'upstream/current' into current

marcushoff committed:
Merge remote-tracking branch 'upstream/current' into current

c-po commented:
Thank you for the PR, could you please exclude .github/pull.yml from this PR?

We also try to keep a common style on our commit messages (https://docs.vyos.io/en/latest/contributing/development.html#prepare-patch-commit) - it would be nice if you could change your commit message, else I‘d do it when squashing it, but the pull.yml should still not be part of the PR.

Happy to merge afterwards.

marcushoff committed:
openvpn: T2957: 
Status file directory for show command was wrong, resulting in no 
output.
Now points to '/var/run/openvpn/{}.status'

marcushoff commented:
Is that better?
